pommerman/utility.py

```diff
@@ -85,11 +86,25 @@ def make(size, num_rigid, num_wood):
         # Set the players down. Exclude them from coordinates.
         # Agent0 is in top left. Agent1 is in bottom left.
         # Agent2 is in bottom right. Agent 3 is in top right.
-        board[1, 1] = constants.Item.Agent0.value
-        board[size - 2, 1] = constants.Item.Agent1.value
-        board[size - 2, size - 2] = constants.Item.Agent2.value
-        board[1, size - 2] = constants.Item.Agent3.value
-        agents = [(1, 1), (size - 2, 1), (1, size - 2), (size - 2, size - 2)]
+        assert (num_agents % 2 == 0)
+
+        if num_agents == 2:
+            agent_positions = {
+                (1, 1): constants.Item.Agent0.value,
+                (size - 2, size - 2): constants.Item.Agent1.value,
+            }
+        else:
+            agent_positions = {
+                (1, 1): constants.Item.Agent0.value,
+                (size - 2, 1): constants.Item.Agent1.value,
+                (size - 2, size - 2): constants.Item.Agent2.value,
+                (1, size - 2): constants.Item.Agent3.value
+            }
+
+        for pos, value in agent_positions.items():
+            board[pos[0]][pos[1]] = value
+
+        agents = [key for key, _ in agent_positions.items()]
```

Comment:
What is the purpose of this line?

Reply:
Cause I use a dict to manage the different configurations of positions of agents.

Now I think this way is a little unnecessary because nothing becomes simpler and just assigning values to board and agents in if-else can already do the job.